src/interfaces/correction-interfaces/correct-name-interface.ts

```diff
@@ -9,7 +9,27 @@ export interface CorrectNameOptionIF {
 }
 
 export interface NrCorrectionIF {
+  legalType: string,
```

Comment:
Can we type this (and status and entity_type_cd below)?

(It may be that we can't because we don't know what the API sends us. But at some point we should then convert string -> type for our use.)

Reply:
kk, i'll check it out!

Reply:
I believe we can, updated!